mlrun/__init__.py

```diff
@@ -160,3 +160,81 @@ def get_sample_path(subpath=""):
     if subpath:
         samples_path = path.join(samples_path, subpath.lstrip("/"))
     return samples_path
+
+
+def env_from_file(env_file, to_dict=False, set_env=True):
+    """Read and set and/or return environment variables from a file
+    the env file should have lines in the form KEY=VALUE, comment line start with "#"
+
+    example file::
+
+        # this is an env file
+        MLRUN_DBPATH=https://mlrun-api.default-tenant.app.xxx.iguazio-cd1.com
+        V3IO_USERNAME=admin
+        V3IO_API=https://webapi.default-tenant.app.xxx.iguazio-cd1.com
+        V3IO_ACCESS_KEY=MYKEY123
+        AWS_ACCESS_KEY_ID-XXXX
+        AWS_SECRET_ACCESS_KEY=YYYY
+
+    usage::
+
+        # set the env vars from a file + return the results as a dict
+        env_dict = mlrun.env_from_file(env_path, to_dict=True)
+
+    :param env_file: path/url to env file
+    :param to_dict:  set to True to return the env as a dict
+    :param set_env:  set to False to skip updating the current OS env
+    :return: None or env dict
+    """
+    env_vars = {}
+    body = get_dataitem(env_file).get(encoding="utf-8")
+    for line in body.splitlines():
+        if line.startswith("#") or not line.strip():
+            continue
+        key, value = line.strip().split("=", 1)
```

Comment:
Should check that `key` has any value, otherwise lines like `=something` will end up with an empty environment variable. It's perfectly valid for `value` to be empty, but I don't see any sense in having an empty key.

Reply:
ok